www/data/docs_sidebar.yml

```diff
@@ -4,15 +4,50 @@ sidebar_links:
   links:
   - title: Docs Home
     link: "/docs"
-  - title: Using Habitat
+  - title: Install Habitat
+    link: "/docs/install-habitat"
+    sub_links:
+    - title: Download the Habitat CLI
+      link: "/docs/install-habitat/#install-habitat"
+    - title: Configure Your Workstation
+      link: "/docs/install-habitat/#configure-workstation"
+    - title: Download and Install FAQ
+      link: "/docs/install-habitat/#install-faq"
+  - title: Developing Packages
+    link: "/docs/developing-packages"
+    sub_links:
+    - title: Writing a Plan.sh
+      link: "/docs/developing-packages/#write-plans"
+    - title: Config Templates
+      link: "/docs/developing-packages/#add-configuration"
+    - title: Binds and Exports
+      link: "/docs/developing-packages/#pkg-binds"
+    - title: Composite Packages
+      link: "/docs/developing-packages/#composite-pkgs"
+    - title: Plan Builds
+      link: "/docs/developing-packages/#plan-builds"
+    - title: Troubleshooting Builds
+      link: "/docs/developing-packages/#debug-builds"
+    - title: Package Export Formats
+      link: "/docs/developing-packages/#pkg-exports"
+  - title: Working with Builder
+    link: "/docs/using-builder"
+    sub_links:
+    - title: Create an Account
+      link: "/docs/using-builder/#builder-account"
+    - title: Create an Origin
+      link: "/docs/using-builder/#builder-origin"
+    - title: Upload and Promote
+      link: "/docs/using-builder/#sharing-pkgs"
+    - title: Using Multiple Plans
+      link: "/docs/using-builder/#multiple-plans-builder"
+    - title: Set up Automated Builds
+      link: "/docs/using-builder/#builder-builds"
```

Comment:
This link doesn't seem to work -- I think it should be `automated-builds` -- or actually, since it seems to be linked-to from various places, the link `name` should be changed to `builder-builds`.